docs/cog_guides/cleanup.rst

```diff
@@ -0,0 +1,296 @@
+.. _cleanup:
+
+=======
+Cleanup
+=======
+
+This is the cog guide for the cleanup cog. You will
+find detailed docs about usage and commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this::
+
+        [p]load cleanup
+
+.. _cleanup-usage:
+
+-----
+Usage
+-----
+
+This cog contains commands used for "cleaning up" (deleting) messages.
+
+This is designed as a moderator tool and offers many convenient use cases.
+All cleanup commands only apply to the channel the command is executed in.
+
+Messages older than two weeks cannot be mass deleted.
+This is a limitation of the API.
+
+
+.. _cleanup-commands:
+
+--------
+Commands
+--------
+
+.. _cleanup-command-cleanup:
+
+^^^^^^^
+cleanup
+^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cleanup 
+
+**Description**
+
+Base command for deleting messages.
+
+.. _cleanup-command-cleanup-after:
+
+"""""""""""""
+cleanup after
+"""""""""""""
+
+.. note:: |mod-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cleanup after <message_id> [delete_pinned=False]
+
+**Description**
+
+Delete all messages after a specified message.
+
+To get a message id, enable developer mode in Discord's
+settings, 'appearance' tab. Then right click a message
+and copy its id.
+
+**Arguments:**
+
+- ``<message_id>`` The id of the message to cleanup after. This message won't be deleted.
+- ``<delete_pinned>`` Whether to delete pinned messages or not. Defaults to False
+
+.. _cleanup-command-cleanup-before:
+
+""""""""""""""
+cleanup before
+""""""""""""""
+
+.. note:: |mod-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cleanup before <message_id> <number> [delete_pinned=False]
+
+**Description**
+
+Deletes X messages before specified message.
```

Comment:
```suggestion
Deletes X messages before the specified message.
```
(Although I know this isn't an issue directly from you, it should get changed here and in [p]cleanup before's docstring)

Reply:
Fixed with commit